Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedChangeStreamOperations.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import MongoSwiftSync
+
+struct CreateChangeStream: UnifiedOperationProtocol {
```

Comment:
this is basically just a `watch` operation, but the unified runner is fairly adamant about using this name instead because some drivers don't actually send the aggregate command when you call `watch` (we do).

so we could have reused the old `watch`, however as written it has no properties stored at all, so I figured I'd just go ahead and add the better-named version now.